UPDATING.md

```diff
@@ -69,61 +69,86 @@ More tips can be found in the guide:
+You can check other options by running the command `airflow config --help`
+
+For a complete list of updated CLI commands, see https://airflow.apache.org/cli.html.
+
+You can learn about the commands by running ``airflow --help``. For example to get help about the ``celery`` group command,
+you have to run the help command: ``airflow celery --help``.
+
+| Old command                 | New command                        |     Group          |
+|-----------------------------|------------------------------------|--------------------|
+| ``airflow worker``          | ``airflow celery worker``          |    ``celery``      |
+| ``airflow flower``          | ``airflow celery flower``          |    ``celery``      |
+| ``airflow trigger_dag``     | ``airflow dags trigger``           |    ``dags``        |
+| ``airflow delete_dag``      | ``airflow dags delete``            |    ``dags``        |
+| ``airflow show_dag``        | ``airflow dags show``              |    ``dags``        |
+| ``airflow list_dag``        | ``airflow dags list``              |    ``dags``        |
+| ``airflow dag_status``      | ``airflow dags status``            |    ``dags``        |
+| ``airflow backfill``        | ``airflow dags backfill``          |    ``dags``        |
+| ``airflow list_dag_runs``   | ``airflow dags list_runs``         |    ``dags``        |
+| ``airflow pause``           | ``airflow dags pause``             |    ``dags``        |
+| ``airflow unpause``         | ``airflow dags unpause``           |    ``dags``        |
+| ``airflow test``            | ``airflow tasks test``             |    ``tasks``       |
+| ``airflow clear``           | ``airflow tasks clear``            |    ``tasks``       |
+| ``airflow list_tasks``      | ``airflow tasks list``             |    ``tasks``       |
+| ``airflow task_failed_deps``| ``airflow tasks failed_deps``      |    ``tasks``       |
+| ``airflow task_state``      | ``airflow tasks state``            |    ``tasks``       |
+| ``airflow run``             | ``airflow tasks run``              |    ``tasks``       |
+| ``airflow render``          | ``airflow tasks render``           |    ``tasks``       |
+| ``airflow initdb``          | ``airflow db init``                |     ``db``         |
+| ``airflow resetdb``         | ``airflow db reset``               |     ``db``         |
+| ``airflow upgradedb``       | ``airflow db upgrade``             |     ``db``         |
+| ``airflow checkdb``         | ``airflow db check``               |     ``db``         |
+| ``airflow shell``           | ``airflow db shell``               |     ``db``         |
+| ``airflow pool``            | ``airflow pools``                  |     ``pools``      |
+| ``airflow create_user``     | ``airflow users create``           |     ``users``      |
+| ``airflow delete_user``     | ``airflow users delete``           |     ``users``      |
+| ``airflow list_users``      | ``airflow users list``             |     ``users``      |
```

Comment:
I still think we should add "legacy" commands in Airflow CLI to print what is the new command and exit with error. This is a better way of "documenting" it than adding UPDATING.md , Most people will not read the UPDATING.md so we should make it as easy and straightforward for the users to get informed about the new command when they run the old one.

Since we do not have an overlap between the old and new commands it should be rather easy to do (now that we have a table with those commands it should be rather easy. 

Do you plan to do it within this PR @ephraimbuddy @mik-laj? Or as a separate, follow-up PR?

Reply:
My mind is that regardless of the message available to the user as an in-program message, we still need to provide documentation that will summarize the changes.  I can't imagine that the user will have to run each command manually to find out what the new name is. 

Similarly, the warning about the change of the import path in the module code is not enough. My belief is that users will want to read the documents to know what to do. Of course, not everyone will do it, but some will want access to this type of documentation.

I think this ticket focuses only on documentation for now. In the next step, we can also add hints as messages in CLI.

> Since we do not have an overlap between the old and new commands it should be rather easy to do (now that we have a table with those commands it should be rather easy.

We have one overlapping command - airflow config.

Reply:
Could you please create a separate ticket and describe your idea?

Reply:
We definitely need to have a description in one place. This is absolutely needed and I never denied it. 

But my point is that it will be far more useful for the users if they do not have to look it up in UPDATING.md if they run an old command. UPDATING.md will be HUGE and  I just try to think as a user using airflow and being used to run familiar commands. 

Giving them suggestion what they can use instead is much better solution than asking them to look it up somewhere (UPDATING.md will not be readily available after you install Airflow - you will have to look it up).

I understand then that it has not been planned in this ticket right?

Reply:
I think this is just a documentation update for now, but it is a step that will allow us to work on your idea as well.

Reply:
Sure. Happy to approve it then.

Reply:
Still it needs a rebase @ephraimbuddy :)

Reply:
I will be happy to implement the ticket when raised.

Reply:
Raised already :) #10109